cmd/csaf_aggregator/indices.go

```diff
@@ -166,10 +166,11 @@ func (w *worker) writeROLIE(label string, summaries []summary) error {
 			Titel:     s.summary.Title,
 			Published: csaf.TimeStamp(s.summary.InitialReleaseDate),
 			Updated:   csaf.TimeStamp(s.summary.CurrentReleaseDate),
-			Link: []csaf.Link{{
-				Rel:  "self",
-				HRef: csafURL,
-			}},
+			Link: []csaf.Link{
+				{Rel: "self", HRef: csafURL},
+				{Rel: "hash", HRef: csafURL + ".sha512"},
```

Comment:
As we compute the `sha256` as well, is there a reason for not adding it?

Reply:
No particular reason ... I just saw the .sha512 in the example linked in #74.
We can add .sha256 as well. Also as rel "hash"?

Reply:
Yes.

Reply:
[x] done

Reply:
Perfect.